api/routes/api_routes.py

```diff
@@ -106,17 +99,19 @@ def get_insurance(id):
 def all_insurance():
     return homeinsurancecontroller.all_insurance()
 
-@api.route('/api/v1/home-insurance/<id>', methods=['PUT'])
-def update_insurance(id):
-    return homeinsurancecontroller.update_insurance(id)
+# @api.route('/api/v1/home-insurance', methods=['POST'])
```

Comment:
I see we are commenting these routes until Iteration2 - where we have admin authorization on these routes